src/package/sign/sign.proj

```diff
@@ -131,7 +131,6 @@
 
       <!-- DataCollector related dlls -->
       <AssembliesToSign Include="$(ArtifactsDirectory)Microsoft.IntelliTrace.Core.dll" />
-      <AssembliesToSign Include="$(ArtifactsDirectory)Microsoft.VisualStudio.Coverage.Analysis.dll" />
       <AssembliesToSign Include="$(ArtifactsDirectory)Microsoft.VisualStudio.Coverage.Interop.dll" />
```

Comment:
You can remove Microsoft.VisualStudio.Coverage.Interop.dll as well,

Reply:
Done.